node/core/candidate-validation/src/lib.rs

```diff
@@ -60,6 +60,12 @@ mod tests;
 
 const LOG_TARGET: &'static str = "parachain::candidate-validation";
 
+/// The amount of time to wait before retrying after an AmbiguousWorkerDeath validation error.
+#[cfg(not(test))]
+const PVF_EXECUTION_RETRY_DELAY: Duration = Duration::from_secs(1);
```

Comment:
We use 3 secs delay for spawning workers on failure. I don't think there's exact science behind this number, but at least it's more consistent this way:
```suggestion
const PVF_EXECUTION_RETRY_DELAY: Duration = Duration::from_secs(3);
```

Reply:
I don't have a strong opinion, both values have trade-offs. Higher value: More likely transient condition is over already, lower value: More likely we can do the validation in a timely manner.

Reply:
> > More likely we can do the validation in a timely manner.

You mean for approval-voting specifically (no-shows)? Yes, this is the only case, but it's better be a know-show than to initiate a dispute wrongly.